app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ArticlesController < ApplicationController
+  def index
+    @article = Article.all
+  end
+
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+    if @article.save
+      flash[:success] = "Article was successfully created."
+      redirect_to @article
+    else
+      flash[:error] = "Please enter valid amount"
```

Comment:
Rephrase the error message. No amounts here :)

Reply:
yup good catch